lib/matplotlib/axes/_axes.py

```diff
@@ -7681,7 +7681,7 @@ def spy(self, Z, precision=0, marker=None, markersize=None,
             For the marker style, you can pass any `.Line2D` property except
             for *linestyle*:
 
-            %(Line2D)s
+        %(Line2D)s
```

Comment:
Just a comment/question.  Would anyone freak out if I put in a PR that renamed this `_Line2D_docstring` - its a real PITA trying to find where this is defined by doing a global search.